package.json

```diff
@@ -8,12 +8,12 @@
         "eslint-plugin-promise": "^4.2.1",
         "eslint-plugin-react": "^7.18.3",
         "eslint-plugin-standard": "^4.0.1",
-        "expo": "^47.0.0",
+        "expo": "^48.0.0",
         "jest": "^26.6.3",
         "jest-expo": "^44.0.1",
```

Comment:
v48 of jest-expo is also released, do you think this is a good time to update it, too?